layouts/search.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Why don't we append `.erb` on these files? It would make them more likely to syntax highlight correctly in whatever editor you're using, including github web.

Reply:
Fair point. Would you mind opening an issue for this?

Reply:
Opened #549